src/nvim/ex_cmds.c

```diff
@@ -5046,6 +5008,46 @@ int find_help_tags(char_u *arg, int *num_matches, char_u ***matches, int keep_la
   return OK;
 }
 
+// Called when starting to edit a buffer for a help file.
+static void prepare_help_buffer(void)
```

Comment:
Since you changed the comment style already, you can use `///` here for Doxygen.